bakerydemo/templates/breads/bread_page.html

```diff
@@ -2,24 +2,131 @@
 {% load wagtailimages_tags %}
 
 {% block content %}
-    <h1>{{ page.title }}</h1>
-    <figure>
-      {% image self.image fill-600x600 %}
-    </figure>
-
-    <p>{{ page.origin }}</p>
-    <p>{{ page.bread_type }}</p>
-    {% with ingredients=page.ingredients.all %}
-        {% if ingredients %}
-            <h3>Ingredients:</h3>
-            <ul>
-                {% for ingredient in ingredients %}
-                    <li style="display: inline">
-                        {{ ingredient.name }}
-                    </li>
-                {% endfor %}
-            </ul>
-        {% endif %}
-    {% endwith %}
-    {{ page.body }}
-{% endblock content %}
+<div class="container bread-detail">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="col-md-6">
+                <div class="row">
+                    <h1>{{ page.title }}</h1>
+                    
+                    <figure class="hidden-md-up">
+                        {% image self.image width-600 %}
+                    </figure>
+                    
+                    {% if page.introduction %}
+                        <p class="introduction">
+                            {{ page.introduction }}
+                        </p>
+                    {% endif %}
+
+                    {{ page.body }}
+                </div>
+            </div>
+
+            <div class="col-md-5 col-md-offset-1">
+                <div class="row">
+                    <figure class="hidden-md-down">
+                        {% image self.image width-500 %}
+                    </figure>
+                    <ul class="bread-meta">
+                        {% if page.origin %}
```

Comment:
I think it's slightly confusing to use self.image and page.origin in the same template. Would these both work with `page.foo` ?